cmake/modules/RootConfiguration.cmake

```diff
@@ -574,6 +550,8 @@ else()
   set(hastmvagpu undef)
 endif()
 
+unset(found_stdstringview CACHE)
```

Comment:
Add comment: `// Allow change of CMAKE_CXX_STANDARD`?

Reply:
Sure, will do.

Reply:
Done.